scripts/SREP/retry-pruning-cronjob/script.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Delete jobs that fail in openshift-sre-pruning namespace.
+
+set -x
+set -e
+set -o nounset
+set -o pipefail
+
+readonly PRUNING_NS=openshift-sre-pruning
+FailedJobs=()
+
+detect_job() {
+  echo "INFO: finding jobs producing error"
+  for current_ns in ${PRUNING_NS}; do
+    readarray -t Jobs < <(oc get job -n "$current_ns" | awk '{print $1}') 
+    Jobs=("${Jobs[@]:1}")
+    for job in "${Jobs[@]}"; do
```

Comment:
Read the jobs in a loop should work, but it is kind of less efficiency, is it possible we use the oc built-in function to get the failed jobs directly?